### DISCUSSION TITLE
try reverse proxy https://pubs.acs.org get this "This web property is not accessible via this address"

### DISCUSSION THREAD

**Discussion by cgklklcg** [dotnet/yarp#1959] [Q&A] [answered]
I'm try to use Yarp as reverse proxy for website  https://pubs.acs.org. 
Then i get this 

"This web property is not accessible via this address."

It seems that cloudflare don't allow reverse proxy for pubs.acs.org.
Does it possible to solve this problem?

**Comment by Tratcher** [comment, marked as answer]
Can you share how you configured YARP?

**Reply by Tratcher** [reply]
This config worked fine:
```json
{
  "Urls": "http://localhost:5000;https://localhost:5001",
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "Microsoft": "Warning",
      "Microsoft.Hosting.Lifetime": "Information"
    }
  },
  "AllowedHosts": "*",
  "ReverseProxy": {
    "Routes": {
      "minimumroute": {
        "ClusterId": "minimumcluster",
        "Match": {
          "Path": "{**catch-all}"
        }
      }
    },
    "Clusters": {
      "minimumcluster": {
        "Destinations": {
          "destination1": {
            "Address": "https://pubs.acs.org/"
          }
        }
      }
    }
  }
}
```

```C#
var builder = WebApplication.CreateBuilder(args);

builder.Services.AddReverseProxy()
    .LoadFromConfig(builder.Configuration.GetSection("ReverseProxy"));

var app = builder.Build();

app.MapReverseProxy();

app.Run();
```

**Reply by cgklklcg** [reply]
thanks so much for reply,
this is my configured


```json
{
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "Microsoft.AspNetCore": "Warning"
    }
  },
  "AllowedHosts": "*",
  "ReverseProxy": {
    "Routes": {
      "route1": {
        "ClusterId": "cluster1",
        "Match": {
          "Path": "{**catch-all}"
        }
      }
    },
    "Clusters": {
      "cluster1": {
        "Destinations": {
          "destination1": {
            "Address": "https://pubs.acs.org/"
          }
        }
      }
    }
  }
}
```
```c#
var builder = WebApplication.CreateBuilder(args);
builder.Services.AddReverseProxy()
    .LoadFromConfig(builder.Configuration.GetSection("ReverseProxy"));
var app = builder.Build();
app.MapReverseProxy();
app.Run();
```

and i try the  configured you supply
I get same "This web property is not accessible via this address."

I guess This config worked fine for you maybe when you access pubs.acs.org, don't visit cloudflare first? you can directly access pubs.acs.org?

Thanks again for your reply!

**Reply by karelz** [reply]
It looks like environmental issue on your side.
Can you try simple C# HelloWorld app that just creates HttpClient and issues a GET request to your destination ("https://pubs.acs.org/")?

**Reply by cgklklcg** [reply]
thanks for reply。 finally I know why。
it was because cloudflare detective your access and stop it 。
check this article [How To Bypass Cloudflare in 2023](https://scrapeops.io/web-scraping-playbook/how-to-bypass-cloudflare/) and you can find answer。
